sdk/spring/spring-cloud-azure-stream-binder-servicebus/src/main/java/com/azure/spring/cloud/stream/binder/servicebus/ServiceBusMessageChannelBinder.java

```diff
@@ -137,10 +136,10 @@ protected MessageProducer createConsumerEndpoint(ConsumerDestination destination
                     buildCheckpointConfig(properties));
         }
         inboundAdapter.setBeanFactory(getBeanFactory());
-        String instrumentationId = Instrumentation.buildId(CONSUMER, destination.getName() + "/" + (!INVALID_SUBSCRIPTION.equals(group) ? group : ""));
+        String instrumentationId = Instrumentation.buildId(CONSUMER, destination.getName() + "/" + (!INVALID_GROUP.equals(group) ? group : ""));
```

Comment:
Can we extract a private method for this `(!INVALID_GROUP.equals(group) ? group : "")`